tests/amplicon/test_statistics.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for amplicon statistics module.
+
+Copyright (c) 2023 Pixelgen Technologies AB.
+"""
+
+import numpy as np
+from pixelator.amplicon.statistics import (
+    SequenceQualityStatsCollector,
+    SequenceQualityStats,
+)
+
+
+def test_sequence_quality_stats_collector():
+    _ = SequenceQualityStatsCollector(design_name="D21")
+
+
+def test_sequence_quality_stats_collector_update():
+    rng = np.random.default_rng(seed=1)
+
+    collector = SequenceQualityStatsCollector(design_name="D21")
+    # Make a quality array of the same length as the amplicon design
+    collector.update(rng.normal(30, 5, size=132))
+    collector.update(rng.normal(30, 5, size=132))
```

Comment:
Can you remind me on why doing this twice?

Reply:
I just wanted to make sure it was working when calling `update` multiple times, since it's called for every read when it's used in practice.